e2e/cypress/integration/polis/client-participation/embeds.spec.js

```diff
@@ -0,0 +1,156 @@
+describe('Embedded Conversations', () => {
+  // This test requires overriding client-admin/embed.html with
+  // e2e/cypress/fixtures/html/embeds.html
```

Comment:
thanks for adding the code comment here 👍

Reply:
no prob. maybe we should have a makefile to run some of these support actions more easily outside the CI run 🤔 

Although this specific one should go away, if we feel it's ok to just replace the old embed.html. Just wanted to be non-destructive here in the interest of quick merging :)